README.rst

```diff
@@ -21,6 +21,22 @@
 
 Get coala to lint all your languages in your project with one tool and config!
 
+Introduction
+------------
+
+coala is a language independent analysis toolkit. It empowers developers
+to create rules which a project's code should conform to. coala takes care
+of showing these issues to users in a friendly manner, is versatile and
+can be used in any environment. Patches to automatically fix code will
```

Comment:
until now you used the present, and here you use the future. this is inconsistent. you should use the same tense everywhere: let it be ``Patches to automatically fix code are managed too!``